### PR TITLE
fixed draft bugs and editor

### PR DESCRIPTION
**Pull request by ue**

### CODE REVIEW

File: src/screens/editor/container/editorContainer.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { connect } from 'react-redux';
 import { injectIntl } from 'react-intl';
 import { Alert, AsyncStorage } from 'react-native';
 import ImagePicker from 'react-native-image-crop-picker';
+import { get } from 'lodash';
```
Comment (mistikk) [inline src/screens/editor/container/editorContainer.js]:
```suggestion
import get from 'lodash/get';
```

File: src/screens/editor/screen/editorScreen.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import React, { Component } from 'react';
 import { View } from 'react-native';
 import { injectIntl } from 'react-intl';
+import { get } from 'lodash';
```
Comment (mistikk) [inline src/screens/editor/screen/editorScreen.js]:
```suggestion
import get from 'lodash/get';
```

File: src/screens/editor/container/editorContainer.js (hunk 1 above)
Comment (ue) [reply src/screens/editor/container/editorContainer.js]: bitmoji

**Comment by ue** [comment]
@mistikk updated

**Pull request merged**